docs/ref/reference.rst

```diff
@@ -385,9 +440,17 @@ corresponding to as many implementation strategies.
      been created with ``pg_basebackup`` and is now started, catching-up to
      the primary server.
 
-Currently, ``pg_autoctl create`` doesn't know how to initialize from an already
-running PostgreSQL standby node. In that situation, it is necessary to
-prepare a new secondary system from scratch.
+  4. Initialize a secondary node from an existing data directory
+
+	 When the data directory pointed to by the option ``--pgdata`` or the
+	 environment variable ``PGDATA`` already exists, then pg_auto_failover
+	 verifies that the system identifier matches the one of the other nodes
+	 already existing in the same group.
```

Comment:
This doesn't say anything about what happens if this system identifier matches. I guess it runs `pg_rewind`?

Reply:
Nope, we just setup streaming replication to the current primary and start Postgres.